### PR TITLE
added ResourceTiming parser in CDP Network

### PR DESCRIPTION
**Pull request by adiohana**
- [x] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

**Comment by takeya0x86** [comment]
https://github.com/SeleniumHQ/selenium/blob/a64a19f4dd032b943ed4baa195887f8c944f35d9/java/client/src/org/openqa/selenium/devtools/network/model/Response.java#L454-L456

How about this line?  Is it `fromServiceWorker` ?

File: java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java [hunk 1]
```diff
@@ -54,6 +58,32 @@
 
   private Double receiveHeadersEnd;
 
+  public ResourceTiming(Double requestTime, Double proxyStart, Double proxyEnd,
```
Comment (shs96c) [inline java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java]: For fields that are not optional, prefer `double` to `Double`. That means it's impossible to create an instance in an incorrect state, and it saves you needing to do null checks.

File: java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java [hunk 2]
```diff
@@ -219,4 +249,101 @@ public Double getReceiveHeadersEnd() {
...
+
+    Double pushStart = null;
+
+    Double pushEnd = null;
+
+    Double receiveHeadersEnd = null;
+
+    while (input.hasNext()) {
+      switch (input.nextName()) {
+        case "requestTime":
```
Comment (shs96c) [inline java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java]: Nit: can we please have these in alphabetical order?

File: java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java (hunk 1 above)
Comment (adiohana) [reply java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java]: done

File: java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java (hunk 2 above)
Comment (adiohana) [reply java/client/src/org/openqa/selenium/devtools/network/model/ResourceTiming.java]: done

**Comment by adiohana** [comment]
@shs96c 
changes are done. can we merge this?

**Comment by shs96c** [comment]
A quick skim suggests this is great :) Thank you! Merging now!

**Pull request merged**